src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap

```diff
@@ -1352,461 +1352,462 @@ exports[`Story Page should not show the pop-out timestamp when allowDateStamp is
               class="c0 c1"
               dir="ltr"
             >
-              <main
+              <div
                 class="c2"
                 dir="ltr"
-                role="main"
               >
-                <div
-                  class="c3"
-                >
-                  <h1
-                    class="c4"
-                    id="content"
-                    tabindex="-1"
+                <main>
```

Comment:
have we lost some attributes here @oluoluoxenfree ? role main and the others?

Reply:
@greenc05 I was going off MDN:

> The <main> element has a role of main. Developers should always prefer using the correct semantic HTML element over using ARIA.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Main_role

Reply:
You mean it's lost attributes right?

Reply:
We use both <main role="main">, as it was previously please, main is a HTML5 element and we add the role main attribute in addition for AT that does not support HTML5

Reply:
https://github.com/bbc/simorgh/pull/6110/files#diff-67d9f7cbe750226b69381b45f0c9a81dL197